test/index.js

```diff
@@ -90,7 +95,7 @@ test('getAbi calculates correct Node ABI', function (t) {
 test('getAbi calculates correct Electron ABI', function (t) {
   t.throws(function () { getAbi(undefined, 'electron') })
   t.throws(function () { getAbi(getNextTarget('electron'), 'electron') })
-  t.equal(getAbi('10.0.0-beta.1', 'electron'), '82')
+  t.equal(getAbi('10.0.0', 'electron'), '82')
```

Comment:
I think that the Electron code needs to be updated so that the generated targets are always `x.0.0-beta.1` but the only thing that changes when there's a stable release is the `future` property.

```suggestion
  t.equal(getAbi('10.0.0-beta.1', 'electron'), '82')
  t.equal(getAbi('10.0.0', 'electron'), '82')
```